src/password.py

```
@@ -0,0 +1,36 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Password helpers."""
+
+import secrets
+import string
+
+
+def generate_password(length: int = 12) -> str:
```

I would avoid defining default values (this could silently hide an oversight in using the function)

---

Changed in 9114d0a23081ba1a1e82132fb6219302f39d144a